pom.xml

```diff
@@ -453,6 +453,30 @@
                         <consoleOutput>true</consoleOutput>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-assembly-plugin</artifactId>
+                    <version>2.4</version>
+                    <executions>
+                        <execution>
+                            <id>assembly-hugegraph</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>single</goal>
+                            </goals>
+                            <configuration>
+                                <attach>false</attach>
+                                <appendAssemblyId>false</appendAssemblyId>
+                                <outputDirectory>${top.level.dir}
+                                </outputDirectory>
+                                <descriptor>
+                                    ${assembly.descriptor.dir}/assembly.xml
+                                </descriptor>
```

Comment:
As Maven operates within the context of the current module when executing goals, the paths specified here are relative to the `hugegraph-dist` directory and can correctly resolve the `assembly.descriptor.dir` property declared within `hugegraph-dist`.